.direnv/flake-inputs/079q0fpi94z0x8a4rj3bzhjl8lrxm5xy-source

```diff
@@ -0,0 +1 @@
+/nix/store/079q0fpi94z0x8a4rj3bzhjl8lrxm5xy-source
```

Comment:
should _flake-inputs/*_ be committed?

Reply:
Nope, this whole dir should be ignored :sweat_smile: